main.lua

```diff
@@ -250,6 +253,17 @@ local function start(...)
       options.tls.ca = certs.caCertsDebug
     end
 
+    if argv.args.ca or virgo.config['caProvided'] then
+      if argv.args.ca ~= null then
```

Comment:
It seems like command-line should take precedence, in which case this would be shifted after the `virgo.config` condition.